SR2019/2018-11-19-kit-disclaimer.md

```diff
@@ -0,0 +1,18 @@
+---
+to: Student Robotics 2019, didn't attend kickstart
+subject: Receiving your kit
+---
+
+Hello there!
+
+As you were unable to attend kickstart, we need to ship you your kit. Before this can happen, we need you to sign a disclaimer.
+
+As it's a legal document, please print out and sign the disclaimer form below, and send it back to us. Once we have a release form for you, we'll send out your kit to the address listed below. If this address is incorrect, please let us know.
```

Comment:
The description here suggests posting them back to us (given that the document has been printed), yet the PR description talks about forwarding reply emails. Maybe be more explicit about how you want the forms to be sent back?